platform/pulp/platform/fields.py

```diff
@@ -0,0 +1,80 @@
+"""
```

Comment:
I'm not to keen on it.  Looking at other code, it seems standard to have a separate fields.py and models.py  (While we use a package instead of a module, this is the reason we import models in the __init__.py iirc)